packages/babel-register/test/fixtures/browserify/register.js

```diff
@@ -1,4 +1,4 @@
 process.env.BABEL_DISABLE_CACHE = "true";
-require("@babel/register").default({
```

Comment:
This test did _not_ test `packages/babel-register` but instead it is tested against some random `@babel/register` in `node_modules`.